src/main/java/mate/academy/StringBootIntro/model/Book.java

```diff
@@ -83,7 +83,7 @@ public void setCoverImage(String coverImage) {
 
     @Override
     public String toString() {
-        return "Book = {"
+        return "Book = { "
```

Comment:
Let's use Lombok annotations for generating get and set methods and toString also